src/uuid/builder/uuidutils.rs

```diff
@@ -154,28 +114,24 @@ impl UuidUtils {
     /// # Returns
     ///
     /// * `Option<u64>` - The number of milliseconds since Unix epoch if the UUID version is supported, otherwise None.
-    pub fn get_time(uuid: &uproto_Uuid) -> Option<u64> {
-        if let Some(version) = UuidUtils::get_version(uuid) {
-            match version {
-                Version::TimeOrdered => Uuid::from(uuid.clone())
-                    .get_timestamp()
-                    .map(|time| time.to_rfc4122().0),
-                Version::Uprotocol => {
-                    let uuid = Uuid::from(uuid.clone());
-                    // Re-assemble the original msb (u64)
-                    let uuid_bytes: &[u8; 16] = uuid.as_bytes();
-                    if let Ok(arr) = uuid_bytes[..8].try_into() {
-                        let msb = u64::from_le_bytes(arr);
-                        let time = msb >> 16;
-                        Some(time)
-                    } else {
-                        None
-                    }
+    pub fn get_time(uprotocol_uuid: &uproto_Uuid) -> Option<u64> {
+        let uuid = uuid::Uuid::from(uprotocol_uuid);
+        match Version::from_value(uuid.get_version_num()) {
+            Some(Version::TimeOrdered) => uuid
```

Comment:
I noticed [over here](https://github.com/eclipse-uprotocol/uprotocol-rust/pull/9/commits/092aea461fde0386e43be3c869e3065e7b2cbc7d#diff-dbb7b4c4d32b29138436a6c7cba0b343a3897dc209bf195e879aca12bff89161L67) the `UUIDv6Builder` was removed.

I'm curious if there's a reason that we'd still like to support retrieving the time from a v6 UUID if they're not supported over uProtocol.

Saw also in the PR summary:
> Removed obsolete v6 UUID variant because the uProtocol spec defines the UUIDs as being v8 only.

I guess we'd delete the lines:
```rust
        match Version::from_value(uuid.get_version_num()) {
            Some(Version::TimeOrdered) => uuid
                .get_timestamp()
                .map(|time| time.to_rfc4122().0),
```
if that's the case?

Reply:
thanks for spotting :+1: